src/Lib/ManticoreBackup.php

```diff
@@ -163,7 +163,7 @@ protected static function store(ManticoreClient $client, FileStorage $storage, a
 			);
 
 			$backupPath = $destination['data'] . DIRECTORY_SEPARATOR . $index;
-			$storage->createDir(
+			$storage::createDir(
```

Comment:
Static calls should use ClassName instead of dynamic variable value. Consider using `FileStorage::createDir`